.gitignore

```diff
@@ -4,23 +4,18 @@ notes
 out.txt
 
 # Folders created in the build process
+.cache
 build
 build-cache
 bundle
 dist
 dist-test
 dist-build
-
-# http-server executable
-http-server/http-server
-
-# gatsby related files
-.cache
 node_modules
 public
 
-# ignore the build.js for secure-upload
-modules/secure-upload/build.js
+# http-server executable
+http-server/http-server
```

Comment:
Why do we want to include http-server executable in the repo? You can always run it with `npx http-server`.

Reply:
users that run it have much faster loading times, I'm not sure what `npx http-server` does

Reply:
but if it's fetching a package remotely and then running code we probably don't want it, this is supposed to be trustless.